src/plugins/discourse/weights.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+
+import {type NodeWeight} from "../../core/weights";
+import {DEFAULT_TRUST_LEVEL_TO_WEIGHT} from "./createGraph";
+import {type User} from "./fetch";
+
+export function likeWeight(user: ?User): NodeWeight {
+  if (user == null) {
+    return 0;
+  }
+  return _trustLevelWeight(user);
+}
+
+export function _trustLevelWeight({trustLevel}: User): NodeWeight {
```

Comment:
I think it'd be cleaner to just pass in the trust level directly, rather than the user object. It would match the function name too, as in `return _trustLevelWeight(user.trustLevel)`. If it's a weight for a trust level, shouldn't it take a trust level?